astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -404,14 +404,21 @@ def set_axislabel(self, text, minpad=1, **kwargs):
             The axis label text.
         minpad : float, optional
             The padding for the label in terms of axis label font size.
+        fontdict : dict, optional
+            Dictionary of settings to pass to the label
```

Comment:
This description is rather unclear. Can we do with just leaving it in `**kwargs` and then below use:
```
fontdict = kwargs.pop('fontdict', None)
```
Or is it used as a positional argument in any call?

Reply:
Happy to change it